pkgs/applications/editors/emacs-modes/melpa-packages.nix

```diff
@@ -151,6 +151,13 @@ self:
       # missing OCaml
       ocp-indent = markBroken super.ocp-indent;
 
+      orgit =
+        (super.orgit.overrideAttrs (attrs: {
+          # searches for Git at build time
```

Comment:
It is not needed at runtime?

Reply:
afaik it calls only magit-commands and not git directly. Is it maybe a magit input propagation issue?